sdk/identity/identity/src/credentials/deviceCodeCredential.ts

```diff
@@ -177,7 +177,8 @@ export class DeviceCodeCredential implements TokenCredential {
 
       while (tokenResponse === null) {
         try {
-          await delay(deviceCodeResponse.interval * 1000);
+          // Referencing delay from core-http this way for testing purposes.
```

Comment:
Am curious as to why we need this comment. What is being said can be inferred from the one line code that follows. Is there something about generally not wanting to import from core-http and therefore we are adding an explanation for an exception to the rule?

Reply:
I've found several local implementations of `delay` when we could've used core-http. I am not sure how to incentive the use of core-http's delay. I wonder if more clarity could help?

Reply:
> I've found several local implementations of delay when we could've used core-http

I noticed that we deleted the local delay.ts file in this PR. So, this should no longer be a problem for Identity

>  I am not sure how to incentive the use of core-http's delay

You can log an issue for each package where this is implemented and recommend using delay from core-http

> I wonder if more clarity could help

I don't see how the comment can help people make use of core-http's delay any more than the actual code change that you have already made